mobile/src/main/java/rs/readahead/washington/mobile/views/fragment/forms/DraftFormsListFragment.kt

```diff
@@ -0,0 +1,167 @@
+package rs.readahead.washington.mobile.views.fragment.forms
+
+import android.os.Bundle
+import android.view.View
+import android.widget.TextView
+import androidx.lifecycle.Observer
+import androidx.lifecycle.ViewModelProvider
+import androidx.recyclerview.widget.LinearLayoutManager
+import androidx.recyclerview.widget.RecyclerView
+import org.hzontal.shared_ui.bottomsheet.BottomSheetUtils
+import org.hzontal.shared_ui.bottomsheet.BottomSheetUtils.ActionSeleceted
+import org.hzontal.shared_ui.utils.DialogUtils
+import rs.readahead.washington.mobile.MyApplication
+import rs.readahead.washington.mobile.R
+import rs.readahead.washington.mobile.bus.event.ShowFormInstanceEntryEvent
+import rs.readahead.washington.mobile.databinding.FragmentDraftFormsListBinding
+import rs.readahead.washington.mobile.domain.entity.collect.CollectFormInstance
+import rs.readahead.washington.mobile.mvp.contract.ICollectFormInstanceListPresenterContract
+import rs.readahead.washington.mobile.mvp.presenter.CollectFormInstanceListPresenter
+import rs.readahead.washington.mobile.views.adapters.CollectDraftFormInstanceRecycleViewAdapter
+import rs.readahead.washington.mobile.views.base_ui.BaseBindingFragment
+import rs.readahead.washington.mobile.views.interfaces.ISavedFormsInterface
+import timber.log.Timber
+
+class DraftFormsListFragment : BaseBindingFragment<FragmentDraftFormsListBinding>(
+    FragmentDraftFormsListBinding::inflate
+),
+    FormListInterfce, ICollectFormInstanceListPresenterContract.IView, ISavedFormsInterface {
+    var recyclerView: RecyclerView? = null
+    var blankFormsInfo: TextView? = null
+    private var model: SharedFormsViewModel? = null
+    private var adapter: CollectDraftFormInstanceRecycleViewAdapter? = null
+    private var presenter: CollectFormInstanceListPresenter? = null
+    override fun getFormListType(): FormListInterfce.Type {
+        return FormListInterfce.Type.DRAFT
+    }
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        adapter = CollectDraftFormInstanceRecycleViewAdapter(this)
+    }
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+        recyclerView = binding!!.draftFormInstances
+        blankFormsInfo = binding!!.blankDraftFormsInfo
+        val mLayoutManager: RecyclerView.LayoutManager = LinearLayoutManager(activity)
+        recyclerView!!.layoutManager = mLayoutManager
+        recyclerView!!.adapter = adapter
+        model = ViewModelProvider(this).get(
```

Comment:
as we migrated to kotlin we should use `ViewModelProvider`like this `    private val model : SharedFormsViewModel by viewModels()
`